mantle/kola/tests/ignition/security.go

```diff
@@ -61,9 +61,9 @@ func init() {
 			"TLSServe": register.CreateNativeFuncWrap(TLSServe),
 		},
 		Tags: []string{"ignition"},
-		// DO: https://github.com/coreos/bugs/issues/2205
-		// Packet & QEMU: https://github.com/coreos/ignition/issues/645
-		ExcludePlatforms: []string{"do", "packet", "qemu"},
+		// QEMU: No support for multiple QEMU VMs communicating on a private network
+		// https://github.com/coreos/coreos-assembler/commit/bb64c669061dbb27d260e2cad3b16cba5813e40a
```

Comment:
```suggestion
		// QEMU: When we moved to unprivileged QEMU we lost support for multiple QEMU VMs 
		// communicating on a private network. 
		// https://github.com/coreos/coreos-assembler/commit/bb64c669061dbb27d260e2cad3b16cba5813e40a
```
optional suggestion to make it a little more clear.

Reply:
added your suggestion, thanks!